README.md

```diff
@@ -117,7 +140,28 @@ curl -H "Content-Type: application/json" \
 -d @/path/to/json/config/file.json \
 http://localhost:8983/solr/<target-core>/update?commitWithin=1000
 ``` 
+
+#### Types and Kinds of sites
+Sites can be of different types and kinds. The type and kind of the remote site (the one that's not 
+the local site) will determine whether the replication is a push, pull, harvest or both.
```

Comment:
Instead of just "both" it could be "both push and pull" to avoid confusion because of the third term "harvest".